src/dom/scroll.js

```diff
@@ -0,0 +1,164 @@
+/**
+ * @license Copyright (c) 2003-2017, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+/* global Node */
+
+/**
+ * @module utils/dom/scroll
```

Comment:
There are [lots](https://github.com/ckeditor/ckeditor5-utils/pull/174/files#diff-94487ad070262037a0877767f74a71c1) of them.